test/validation/api/ipsec/ipsec.h

```diff
@@ -27,6 +27,8 @@ int ipsec_in_term(void);
 int ipsec_out_term(void);
 
 struct suite_context_s {
+	odp_bool_t reass_ipv4;
+	odp_bool_t reass_ipv6;
```

Comment:
These are not used before ae162b1df, so I think they should be moved there.

Reply:
Missed this. Will address in next version.